va/caa.go

```diff
@@ -213,14 +212,17 @@ func (va *ValidationAuthorityImpl) parallelCAALookup(ctx context.Context, name s
 		// Start the concurrent DNS lookup.
 		wg.Add(1)
 		go func(name string, r *caaResult) {
+			defer wg.Done()
```

Comment:
Optional: we could convert this to `wg.Go()`.

Reply:
wg.Go() requires that its inner function take no arguments, while this function takes two. We could change it to be a closure, but I think keeping it as-is is better.